src/interactive/repl.ts

```diff
@@ -549,6 +550,40 @@ function interrupt() {
 
 // code execution end
 
+async function cdToHere(uri: vscode.Uri) {
+    const uriPath = await getDirUriFsPath(uri)
+    if (uriPath) {
+        g_connection.sendNotification('repl/cd', uriPath)
+    }
+}
+
+async function activateHere(uri: vscode.Uri) {
+    const uriPath = await getDirUriFsPath(uri)
+    if (uriPath) {
+        g_connection.sendNotification('repl/activateProject', uriPath)
+        switchEnvToPath(uriPath, true)
+    }
+}
+
+async function getDirUriFsPath(uri: vscode.Uri) {
```

Comment:
```suggestion
async function getDirUriFsPath(uri: vscode.Uri | undefined) {
```

for preparation for future strict mode ?